modules/core/src/main/scala/org/scalasteward/core/git/CommitMsg.scala

```diff
@@ -52,10 +52,21 @@ object CommitMsg {
           .replace("${branchName}", baseBranch.map(_.name).orEmpty)
         CommitMsg(title = title)
       },
-      g =>
-        CommitMsg(title =
-          g.title.getOrElse(s"Update for group ${g.name}") +
-            baseBranch.fold("")(branch => s" in ${branch.name}")
-        )
+      g => {
+        val artifactsWithVersions = g.updates
```

Comment:
**issue:** This won't affect commit messages for individual updates, those are created with the previous function in this method (first method of `update.on`).

[These are the places where this method is called](https://github.com/search?q=repo%3Ascala-steward-org%2Fscala-steward+%22replaceVariables%22&type=code). As you can see, every call is done for an `Update.Single`, except for [this one](https://github.com/scala-steward-org/scala-steward/blob/5a643140dbbbf11548335997bfb139f13362a238/modules/core/src/main/scala/org/scalasteward/core/forge/data/NewPullRequestData.scala#L252) which creates the pull request title. This change will create PR titles appending updates information until GitHub truncates the title, which is something we don't want. So I would suggest reverting this part.

If commit messages aren't descriptive enough for you, maybe you can change it in your codebase using [`commits.message`](https://github.com/scala-steward-org/scala-steward/blob/609104a04cd536418ebcda553e7f48177a5264ae/docs/repo-specific-configuration.md?plain=1#L126) configuration.

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>

Reply:
Reverting this would no longer disclose the artifacts and versions for an update. Which is something I like to see.  After a squash merge, the individual commit messages are gone. 

I propose: Keep the default the same (`s"Update for group ${g.name}") +             baseBranch.fold("")(branch => s" in ${branch.name}"`), but still allow for `${artifactVersions}` to be specified in the grouping configuration through a variable.

Is that okay to you?

Reply:
Yes, I think that would be preferable, although this won't help much unless you have a very small group... since as I mention, GitHub will truncate the title

Reply:
I've not seen the title being truncated yet (in my ecosystem) However, I could do something in the lines of:
`Update artifact-1, ...., artifact-2 to version 5` to make the titles shorter, in case they are too long.